rfcs/roadmap-template.md

```diff
@@ -0,0 +1,73 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# DCP Product Roadmap YYYY
+
+*Change `YYYY` to the target year for the Roadmap such as `2020`*
+
+## Summary
+
+*Describe the highlights of the Roadmap in 2-3 sentences. Consider this section as your "elevator pitch" or "release note" to the community.*
+
+## Author(s)
+
+*Recommended format for Authors:*
+
+ `[Name](mailto:username@example.com)`
+
+*This could be individual authors or the DCP Project Leads mailing list.*
+
+## Shepherd
+*This role MUST be assigned to the Roadmap Shepherd to guide the **Author(s)** through the RFC process.*
+
+*Recommended format for Shepherds:*
+
+ `[Name](mailto:username@example.com)`
+
+## User Survey
+
+*Summarize the overall alignment with the User Survey and include a reference to the Survey*
+
+## Scientific "guardrails"
+
+*Summarize the overall alignment with the Science Roadmap and include a reference to the Roadmap*
+
+## Theme 1
+
+*Provide a detailed overview of the Theme that provides context and information.*
+
+### Objective 1
+
+*Document high-level Objectives that are achieved by the completion of the Theme*
+
+#### User Story
+#### User Value
+
+*Document how this Objective delivers value to specific [HCA Personas](https://docs.google.com/presentation/d/1bsu8q9CzRXv3Y8c1p8NENi3p8KlxC4scp2tJkHO3Jeo/edit#slide=id.g3cf4e46e8f_0_38).*
+
```

Comment:
I believe PLs are to communicate what is the success criteria and demoable thing for the objective. Would this be included here?

Reply:
See - 1ff23013d309a1072d21e17013090ce0e2603f0e - for most recent updates.

Reply:
@brianraymor -- @TimothyTickle and I discussed this in the PL Channel. We would like to have a short form for the themes in the Roadmap, and a linkout to the long-form of the theme.

For this RFC, it would be best if we keep the short form in this template, which would include for each theme:

Theme (User Scenario), including a link to the long-form of the Theme
Objectives (Goals, in the context of as user value if possible)
Accountable (name and email)

Notifying @CaraLMason as well for visibility

Reply:
And are you planning to add the _linkout to the long-form_ at a later date?

Reply:
Yes. In our current roadmap, we have not put the necessary detailed information. I think in the short term for the **_critical_** items we can start working on the long-form themes and determine what parts are required.

Reply:
@tj and @TimothyTickle - Considering this further, the _linkout_ to the long-form of the Themes does not make sense to me. It makes it more difficult for the HCA community to review the roadmap (similar to my comments about a separate strategy document). Perhaps I'm missing context. Could offer further rationale during Oversight Review? In the interim, I'm leaving the template unchanged.

Reply:
My comment is resolved, great examples.